awesomerc.lua

```diff
@@ -481,9 +482,11 @@ client.connect_signal("manage", function (c)
         -- Prevent clients from being unreachable after screen count changes.
         awful.placement.no_offscreen(c)
     end
+end)
 
-    local titlebars_enabled = true
-    if titlebars_enabled and (c.type == "normal" or c.type == "dialog") then
+-- Add a titlebar if titlebars_enabled is set to true in the rules.
+client.connect_signal("request::titlebars", function(c)
+    if (c.type == "normal" or c.type == "dialog") then
```

Comment:
Should this type-check also be moved into `awful.rules`? If I'm not mistaken, adding a new rule with `{ rule_any = { type = { "normal", "dialog" } }, properties = { titlebars_enabled = true } }` should work (and of course, I did not try this).